src/ExNavigationBar.js

```diff
@@ -327,6 +340,21 @@ export default class ExNavigationBar extends PureComponent {
     }
 
     const pointerEvents = offset !== 0 || isStale ? 'none' : 'box-none';
+
+    if (name === 'background') {
+      return (
+        <View
+          pointerEvents={pointerEvents}
```

Comment:
Probably should've `pointerEvents='none'`? Why does an element inside background need to respond to touch (when `pointerEvents='box-none'`)